src/devices/Card/LogInfo.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+namespace Iot.Device.Card
+{
+    /// <summary>
+    /// The log level
+    /// </summary>
+    public enum LogLevel
+    {
+        None = 0,
+        Info = 1,
+        Debug = 2
+    }
+
+    /// <summary>
+    /// Output where to log the information
+    /// </summary>
+    [Flags]
+    public enum LogTo
+    {
+        Console = 0b0000_00001,
+        Debug = 0b0000_0010
+    }
+
+    /// <summary>
+    /// Simple log class to help in debugging the communication
+    /// between the PN532 and the host
+    /// </summary>
+    public class LogInfo
+    {
+        public static LogLevel LogLevel { get; set; }
+
+        public static LogTo LogTo { get; set; } = LogTo.Console;
```

Comment:
I still think this should use TextWriter - you can even make the abstraction similar:

```csharp
static class LogTo
{
  public static readonly TextWriter Console = Console.Out;
  // debug actually doesn't have TextWriter but you can either create one or just not have this option
}
```

Reply:
How to log both in the Console and in the Debug then? That's what I'm missing with this solution.

Reply:
Some example:

```csharp
using System;
using System.Diagnostics;
using System.IO;
using System.Text;

namespace ConsoleApp196
{
    class MultiTextWriter : TextWriter
    {
        TextWriter[] _writers;
        public MultiTextWriter(params TextWriter[] writers)
        {
            _writers = writers;
        }

        public override Encoding Encoding => Encoding.UTF8;

        public override void Write(char value)
        {
            foreach (var writer in _writers)
            {
                writer.Write(value);
            }
        }
    }

    class DebugWriter : TextWriter
    {
        public override Encoding Encoding => Encoding.UTF8;

        public override void Write(char value)
        {
            Debug.Write(value);
        }
    }

    class Program
    {
        static void Main(string[] args)
        {
            TextWriter mw = new MultiTextWriter(Console.Out, new DebugWriter());

            mw.WriteLine("TEST");
        }
    }
}
```